package.json

```diff
@@ -42,7 +42,8 @@
   "dependencies": {
     "is-react-prop": "^0.0.3",
     "jss": "^7.1.0",
-    "jss-preset-default": "^2.0.0"
+    "jss-preset-default": "^2.0.0",
+    "raf": "^3.3.0"
```

Comment:
We can't use just `window.requestAnimationFrame`, because we run this code not only in the browser (tests, ssr) - so we need polyfill

But we can make it customizable